Formula/python3.rb

```diff
@@ -310,7 +310,22 @@ def post_install
   end
 
   def xy
-    version.to_s.slice(/(3\.\d)/) || "3.6"
```

Comment:
Can't we use the version reported by upstream when you invoke `python --version`?